---
authors: 
- Meethila Gade
- Dulcie Lai
- Erin Heinzen
abstract: ""
address:
  city: Nashville
  country: United States
  postcode: "94305"
  region: CA
  street: 450 Serra Mall
all_day: false
date: "2022-12-01T13:00:00Z"
date_end: "2022-12-01T15:00:00Z"
event: American Epilepsy Society Annual Meeting 2022
event_url: https://cms.aesnet.org/abstractslisting/determining-single-cell-transcriptomic-signatures-in-mosaic-epileptogenic-human-brain-tissue
featured: false
image:
  picture: content/poster/AES 2022/featured.jpeg
  caption: 'AES 2022 Annual Meeting'
  focal_point: Right
links:
projects:
- example
publication: At  *AES 2022 Annual Meeting*
publication_short: At *AES 2022*
publishDate: "2022-12-01T00:00:00Z"
slides: ""
summary: ""
tags: [Somatic mutations, Epilepsy]
title: Determining Single-cell Transcriptomic Signatures in Mosaic Epileptogenic Human Brain Tissue
url_code: ""
url_poster: Poster_AES.pdf
url_slides: ""
url_video: ""
---
Abstract: 
Rationale: Somatic variants acquired post-zygotically during embryonic cortical development are known to contribute to epilepsy-associated cortical brain malformations. The mosaic brain tissue surgically resected from affected individuals offers a unique opportunity to determine the burden of mutation across cell types, as well as the transcriptional effects of the mutation in specific cell types. However, simultaneous assessments of DNA and RNA from single cells required to harness this potential remain technically challenging. Here we describe a novel method that uses parallel single-cell genotyping and comprehensive transcriptomic profiling to study the cellular heterogeneity in mosaic epileptogenic brain tissue.

  Methods: Adapted from the G&Tseq protocol,1 we developed and optimized a method to simultaneously isolate DNA and RNA from single cells to identify the transcriptomic signature in single nuclei with and without the mutation (Figure 1). We benchmarked the protocol by comparing nuclei isolated from surgically resected brain tissue of a patient with hemimegalencephaly harboring a pathogenic somatic variant in PIK3CA (E545K) to an age-matched control sample. A fluorescently-labeled antibody against NeuN was used to isolate neuronal nuclei from the tissue specimens using fluorescence-activated nuclear sorting. Bulk (n=20,000 nuclei/total nuclei) and single nuclei (n=44 NeuN+ nuclei/sample) were collected for the case and control sample. RNA and DNA were isolated from single nuclei and bulk nuclei preparations. RNAseq was performed at a sequencing depth of 15 million (mil) reads/sample for NeuN- and NeuN+ bulk nuclei collections and for the 88 NeuN+ single nuclei from the case and control samples. Genotyping of amplified DNA was used to identify the mutation-positive cells using droplet digital PCR and a custom-designed TaqMan genotyping assay.

  Results: We saw a variant allele fraction of 30% (NeuN+) and 35% (NeuN-) in bulk nuclei collections from the mosaic PIK3CA brain tissue sample, which correlated with that seen in single nuclei. RNA from bulk nuclei populations consistently yielded ~10 mil uniquely mapped reads (UMR). We observed a transcriptomic signature consistent with an upregulation of the mTOR signaling pathway in both NeuN- and NeuN+ bulk nuclei in cases compared to controls. In addition, we found a transcriptomic signature associated with PIK3CA variants specific to NeuN+ nuclei in the CREB phosphorylation pathway. On average, we obtained ~5 mil UMRs per nuclei (80,000-13 mil, with >50% of cells achieving > than 5 mil UMRs per cell). Analysis of genotype-informed single nuclei RNAseq is ongoing.

  Conclusions: This optimized approach for simultaneous single-cell genotyping and comprehensive cDNA sequencing will be a valuable tool for identifying cell type specific transcriptional profiles associated with recently identified epilepsy-associated mosaic mutations in human brain tissue and may guide us to novel therapeutic targets for treating intractable seizures.