---
abstract: ""
author_notes: ""
authors:
- admin
- Dulcie Lai
- Erin Heinzen
date: "2022-11-30T00:00:00Z"
doi: ""
featured: false
image:
  caption: ""
  focal_point: ""
  preview_only: false
projects:
- example
publication: At  *ASHG 2022 Annual Meeting*
publication_short: At *ASHG 2022*
publication_types:
- "2"
publishDate: "2022-11-30T00:00:00Z"
slides: ""
summary: ""
tags: [somatic mutations, epilepsy]
title: Leveraging Mosaic Epileptogenic Human Brain Tissue to Study Cell-type Specific Transcriptional Changes Associated with a Pathogenic PIK3CA Variant
url_code: ""
url_dataset: ""
url_pdf: https://www.ashg.org/wp-content/uploads/2022/09/ASHG2022-PosterAbstracts.pdf
url_poster: Poster_ASHG.pdf
url_project: ""
url_slides: ""
url_source: ""
url_video: ""
---
Abstract: Post-zygotically acquired mutations that arise during embryonic cortical development can lead to abnormal neuron morphology and/or migration defects, both of which can often result in seizures and developmental disabilities. Somatic mutations that cause abnormal activation of proteins in the PI3K-AKT-mTOR signaling pathway have been implicated in focal cortical dysplasia type 2 and hemimegalencephaly (HMEG). Surgically resected mosaic brain tissue from affected individuals offers a
unique opportunity to determine the burden of mutations and transcriptional effects of the mutations across cell types. To tease apart the transcriptional effects of the mutation within cell types, we modified the G&Tseq protocol to simultaneously perform single-cell genotyping and transcriptomic profiling to identify the comprehensive transcriptomic signature of mutation-positive and mutation-negative nuclei.
We benchmarked the protocol by comparing nuclei isolated from surgically resected brain tissue of a patient with HMEG harboring a pathogenic somatic variant in PIK3CA (E545K) to an age-matched control. We used a fluorescently labeled NeuN antibody to sort neuronal and non-neuronal nuclei for bulk
(n=20,000 nuclei/total nuclei) and single-cell (n=44 NeuN+ nuclei/sample) preparations. RNA extracted from bulk (NeuN+ and NeuN-) and single NeuN+ nuclei for both cases and controls was sequenced at a depth of 15 million (MM) reads/sample. In parallel, DNA was genotyped in mutation-positive bulk and
single nuclei using droplet digital PCR and TaqMan genotyping assay, respectively. A variant allele fraction of 30% (NeuN+) and 35% (NeuN-) was observed in bulk samples from the mosaic PIK3CA tissue, which correlated with that observed in single nuclei. RNA from bulk nuclei populations consistently yielded ~10 MM uniquely mapped reads (UMR) while RNA from single nuclei
yielded ~5-10 MM UMR. Transcriptomic analysis of bulk nuclei (NeuN- & NeuN+) and single nuclei (NeuN+) confirmed the activation of the mTOR pathway. We also observed transcriptomic signature associated with PIK3CA variants specific to NeuN+ bulk and single nuclei; namely, the CREB phosphorylation pathway and an upregulation of SPDYE5, a gene implicated in low-grade glioma.
Analyses of genotype-informed single nuclei RNAseq are currently underway. This approach will allow for novel insights into the impact of pathogenic somatic mutations on cell-autonomous and non-cellautonomous functions, high-resolution assessments of cell-type-specific mutation burden, and may help identify antiseizure targets for treating mTOR related intractable seizures.
